infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -245,8 +245,8 @@ public void writeHeader(FileChannel fileChannel, int version, boolean verify) th
                 .build();
 
         ByteBuffer buf = getByteBufferWithMetaData(header);
-        writeByteBuffer(fileChannel, buf);
-        fileChannel.force(true);
+        IOLatencyDetector.write(fileChannel,buf, false, 0);
```

Comment:
By the way, instead of exposing these methods as statics why don't you introduce a new type that wraps FileChannel ? I think that's much cleaner.

Reply:
Will look into this.